mne/viz/topomap.py

```diff
@@ -1620,3 +1620,213 @@ def _prepare_topomap(pos, ax):
+    norm = True if np.min(data) > 0 else False
+    cmap = 'Reds' if norm else 'RdBu_r'
+    contours = params['contours']
+    times = params['times']
+    vmin, vmax = _setup_vmin_vmax(data, None, None, norm)
+
+    pos, outlines = _check_outlines(params['pos'], 'head', None)
+    pos_x = pos[:, 0]
+    pos_y = pos[:, 1]
+
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_frame_on(False)
+    xlim = np.inf, -np.inf,
+    ylim = np.inf, -np.inf,
+    mask_ = np.c_[outlines['mask_pos']]
+    xmin, xmax = (np.min(np.r_[xlim[0], mask_[:, 0]]),
+                  np.max(np.r_[xlim[1], mask_[:, 0]]))
+    ymin, ymax = (np.min(np.r_[ylim[0], mask_[:, 1]]),
+                  np.max(np.r_[ylim[1], mask_[:, 1]]))
+
+    res = 64
+    xi = np.linspace(xmin, xmax, res)
+    yi = np.linspace(ymin, ymax, res)
+    Xi, Yi = np.meshgrid(xi, yi)
+    params['Zis'] = list()
+
+    for frame in params['frames']:
+        Zi = _griddata(pos_x, pos_y, data[:, frame], Xi, Yi)
+        params['Zis'].append(Zi)
+    Zi = params['Zis'][0]
+
+    _, pos = _make_image_mask(outlines, pos, res)
+
+    params.update({'vmin': vmin, 'vmax': vmax, 'Xi': Xi, 'Yi': Yi, 'Zi': Zi,
+                   'outlines': outlines, 'extent': (xmin, xmax, ymin, ymax),
+                   'pos_x': pos_x, 'pos_y': pos_y, 'cmap': cmap})
+    # plot map and countour
+    im = ax.imshow(Zi, cmap=cmap, vmin=vmin, vmax=vmax, origin='lower',
+                   aspect='equal', extent=(xmin, xmax, ymin, ymax),
+                   interpolation='bilinear')
+    plt.colorbar(im, cax=ax_cbar, cmap=cmap)
+    cont = ax.contour(Xi, Yi, Zi, contours, colors='k', linewidths=1)
+
+    patch_ = patches.Ellipse((0, 0),
+                             2 * outlines['clip_radius'][0],
+                             2 * outlines['clip_radius'][1],
+                             clip_on=True,
+                             transform=ax.transData)
+    im.set_clip_path(patch_)
+    text = ax.text(0.45, 0.95, '', transform=ax.transAxes, va='center')
+    params['text'] = text
+    items = [im, text]
+    for col in cont.collections:
+        col.set_clip_path(patch_)
+
+    outlines_ = dict([(k, v) for k, v in outlines.items() if k not in
+                      ['patch', 'autoshrink']])
+    for k, (x, y) in outlines_.items():
+        if 'mask' in k:
+            continue
+        ax.plot(x, y, color='k', linewidth=1, clip_on=False)
+    params['patch'] = patch_
+
+    if params['butterfly']:
+        for idx in range(len(data)):
+            ax_line.plot(times, data[idx], color='k')
+        params['line'], = ax_line.plot([times[0], times[0]],
+                                       ax_line.get_ylim(), color='r')
+        items.append(params['line'])
+
+    return tuple(items) + tuple(cont.collections)
+
+
+def _animate(i, ax, ax_line, params):
+    """Updates animated topomap."""
+    times = params['times']
+    time_idx = params['frames'][i]
+
+    title = '%.1f ms' % (times[time_idx] * 1e3)
```

Comment:
`int` should be enough, no-one does sub milliseconds analysis. (Although... some high frequency traders might be using our great package?)